spec/schemas/AddressBase32.yml

```diff
@@ -1,3 +1,3 @@
 type: string
 description: Address encoded using a 32-character set.
```

Comment:
39 instead of 32

Reply:
Looks correct. It' encoded using [32-charater set](https://en.wikipedia.org/wiki/Base32) - comprised of comprising the twenty-six upper-case letters A–Z, and the digits 2–7 - although the length is 39.